### PR TITLE
Feature/document provider application test account

### PR DESCRIPTION
**Pull request by hinriksteinar**
# \<Description\>

Attach a link to issue if relevant

## What

Finishing the test phase look and feel

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/document-provider-onboarding/src/fields/TestPhase/TestEnvironment/index.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React, { FC, useEffect, useState } from 'react'
+import { FieldBaseProps } from '@island.is/application/core'
+import { Box, Button, Input, Text } from '@island.is/island-ui/core'
+import { FieldDescription } from '@island.is/shared/form-fields'
+import CopyToClipboardInput from '../../DocumentProvicerApplication/Components/CopyToClipboardInput/Index'
+
+const TestEnvironment: FC<FieldBaseProps> = ({ error, field, application }) => {
```
Comment (fjandakornid) [inline libs/application/templates/document-provider-onboarding/src/fields/TestPhase/TestEnvironment/index.tsx:7]: unused variables

File: libs/application/templates/document-provider-onboarding/src/fields/TestPhase/TestEnvironment/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import React, { FC, useEffect, useState } from 'react'
+import { FieldBaseProps } from '@island.is/application/core'
+import { Box, Button, Input, Text } from '@island.is/island-ui/core'
```
Comment (fjandakornid) [inline libs/application/templates/document-provider-onboarding/src/fields/TestPhase/TestEnvironment/index.tsx]: unused variable

File: libs/application/templates/document-provider-onboarding/src/fields/TestPhase/TestEnvironment/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import React, { FC, useEffect, useState } from 'react'
```
Comment (fjandakornid) [inline libs/application/templates/document-provider-onboarding/src/fields/TestPhase/TestEnvironment/index.tsx]: unused variable useEffect

File: libs/application/templates/document-provider-onboarding/src/fields/DocumentProvicerApplication/Review/index.tsx [hunk 4]
```diff
@@ -14,11 +14,7 @@ import {
   Input,
   Text,
```
Comment (fjandakornid) [inline libs/application/templates/document-provider-onboarding/src/fields/DocumentProvicerApplication/Review/index.tsx]: unused variable

File: libs/application/templates/document-provider-onboarding/src/lib/DocumentProviderOnboardingTemplate.ts [hunk 5]
```diff
@@ -56,6 +56,7 @@ const dataSchema = z.object({
   administrativeContact: contact,
   technicalContact: contact,
   helpDesk: helpDeskContact,
+  endPoint: z.string().nonempty(),
```
Comment (fjandakornid) [inline libs/application/templates/document-provider-onboarding/src/lib/DocumentProviderOnboardingTemplate.ts:59]: should this be z.url().nonempty() ?

File: libs/application/templates/document-provider-onboarding/src/fields/DocumentProvicerApplication/Components/CopyToClipboardInput/Index.tsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
+import React, { FC, useRef } from 'react'
+import {
+  Box,
+  Input,
+  Button,
+  ToastContainer,
```
Comment (fjandakornid) [inline libs/application/templates/document-provider-onboarding/src/fields/DocumentProvicerApplication/Components/CopyToClipboardInput/Index.tsx]: unused variables

File: libs/application/templates/document-provider-onboarding/src/fields/TestPhase/ProdEnvironment/index.tsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
+import React, { FC, useEffect, useState } from 'react'
```
Comment (fjandakornid) [inline libs/application/templates/document-provider-onboarding/src/fields/TestPhase/ProdEnvironment/index.tsx]: unused variables

**Comment by fjandakornid** [comment]
Looks good... some unused variables and one comment on the validation of endpoints. I think it should be url since endpoints are always url to some api.

**Review by ssh16 — APPROVED** [review]

File: libs/application/templates/document-provider-onboarding/src/lib/DocumentProviderOnboardingTemplate.ts (hunk 5 above)
Comment (hinriksteinar) [reply libs/application/templates/document-provider-onboarding/src/lib/DocumentProviderOnboardingTemplate.ts:59]: It seems that z.url() does not exist... Can i add it or am i missing something ?

**Review by ssh16 — APPROVED** [review]

**Pull request merged**